### ISSUE TITLE
NPE crash in BaseLifecycleManager on checkLifecycleConfiguration

### ISSUE THREAD

**Issue by kenhuang1120** [smartdevicelink/sdl_java_suite#1783]
### Bug Report
[Summary]
Crash in BaseSdlManager.checkLifecycleConfiguration on line 191. Attempting to call managerListener.managerShouldUpdateLifecycle() when smanagerListener value is null.

##### Reproduction Steps
Unfortunately unable to reproduce at the moment. This crash came from our Crashlytics logs and has only been observed from version 5.1 

##### OS & Version Information
* Android Version: Android 7 to 12
* SDL Android Version: from 5.1
* Testing Against: No tests as this came from Crashlytics logs.

##### Test Case, Sample Code, and / or Example App
None currently available. only crash Log
```
Fatal Exception: java.lang.NullPointerException: Attempt to invoke interface method 'com.smartdevicelink.managers.lifecycle.LifecycleConfigurationUpdate com.smartdevicelink.managers.SdlManagerListener.managerShouldUpdateLifecycle(com.smartdevicelink.proxy.rpc.enums.Language, com.smartdevicelink.proxy.rpc.enums.Language)' on a null object reference
       at com.smartdevicelink.managers.BaseSdlManager.checkLifecycleConfiguration(BaseSdlManager.java:191)
       at com.smartdevicelink.managers.SdlManager$2.run(SdlManager.java:218)
       at android.os.Handler.handleCallback(Handler.java:883)
       at android.os.Handler.dispatchMessage(Handler.java:100)
       at android.os.Looper.loop(Looper.java:359)
       at android.app.ActivityThread.main(ActivityThread.java:7418)
       at java.lang.reflect.Method.invoke(Method.java)
       at com.android.internal.os.RuntimeInit$MethodAndArgsCaller.run(RuntimeInit.java:492)
       at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:935)
```

**Comment by peterkomar** [comment]
I also received similar crash:

**Comment by theresalech** [comment]
Closed via #1786
